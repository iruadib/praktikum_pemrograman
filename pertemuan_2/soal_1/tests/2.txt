John Doe 
150000000 200000 150000