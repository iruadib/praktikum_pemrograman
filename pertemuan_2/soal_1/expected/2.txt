Payslip for Employee
---------------------
Name: John Doe 
Tax (20%): Rp30.000.000,00
Installment: Rp200.000,00
Insurance: Rp150.000,00
Net Salary: Rp119.650.000,00
